app/assets/stylesheets/mtl/layouts/_default.scss

```diff
@@ -21,6 +21,10 @@
   background-color: $mtl-layout-default-header-bg;
   @include clearafter;
 
+  * {
```

Comment:
hmmm in a way yes and no at the same time:

YES&NO: your example would be messed up... BUT your would have full control anyway:
`<div class="grey-text">asdsa <span class="grey-text">didum</span></div>` would work, as all `.COLOR-text` classes override the color with `!important`.

NO: we don't want to have people to use these overrides in the default layout anyway, right? we'd rather extend the functionality.

YES: we could add specific overrides instead of the star... 
NO: ...but does it help the readability/maintainability of the sass layout? I mean we have: navs, titles, icons, dropdowns, other infos and potential custom stuff in each project which I cannot predict. * would cover like 90% of ALL cases. I'd rather prefer having the * there instad of defining each and every possible element that is/could maybe be there.

Food for thought.